### PR TITLE
add info on async jobs

### PR DESCRIPTION
**Pull request by kevashcraft**
Adds a page with an overview and examples of Jobs and how to manage them. Also adds a line to the Summary to link to the new page.

### CODE REVIEW

**Review by terraHDB — CHANGES_REQUESTED** [review]

File: docs/jobs.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Jobs use an asynchronous methodology to account for the potential of a long-running operation. For example, exporting millions of records to S3 could take some time, so that job is started and the id is provided to check on the status.
+
+The job status can be **COMPLETE** or **IN_PROGRESS**.
+
+## Example Job Operations
+
+Example job operations include:
+
+[delete_records_before](https://api.harperdb.io/#8de87e47-73a8-4298-b858-ca75dc5765c2)
```
[the remark this answers is not in the document]
Comment (kevashcraft) [reply docs/jobs.md:23]: added a \n

File: docs/jobs.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+[export_local](https://api.harperdb.io/#49a02517-ada9-4198-b48d-8707db905be0)
```
[the remark this answers is not in the document]
Comment (kevashcraft) [reply docs/jobs.md:25]: added a \n

File: docs/jobs.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Job Summary
+
+Jobs use an asynchronous methodology to account for the potential of a long-running operation. For example, exporting millions of records to S3 could take some time, so that job is started and the id is provided to check on the status.
+
+The job status can be **COMPLETE** or **IN_PROGRESS**.
+
+## Example Job Operations
+
+Example job operations include:
```
[the remark this answers is not in the document]
Comment (kevashcraft) [reply docs/jobs.md:13]: Ah good call. I hadn't realized those were also async. Added the bulk uploads.

File: docs/jobs.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+# Asynchronous Jobs
+
+HarperDB Jobs are asynchronous tasks performed by the Operations API.
+
+## Job Summary
+
+Jobs use an asynchronous methodology to account for the potential of a long-running operation. For example, exporting millions of records to S3 could take some time, so that job is started and the id is provided to check on the status.
```
[the remark this answers is not in the document]
Comment (kevashcraft) [reply docs/jobs.md]: fixed type

**Review by terraHDB — COMMENTED** [review]
👍

**Pull request merged**